README.md

```diff
@@ -1,15 +1,73 @@
-# TH2 common library
+# th2 common library (Java)
+
+## Usage
+
+First things first, you need to import `CommonFactory` class:
+```
+import com.exactpro.th2.common.schema.factory.CommonFactory
+```
+Then you create an instance of imported class, choosing one of the options:
```

Comment:
Then **you will** create an instance of imported class, **by choosing one of the following options**: